### PR TITLE
phi0836.phi002

### PR DESCRIPTION
**Pull request by AlisonBabeu**
Updated the cts.xml
Took first pass at updating and standardizing the headers.
Thanks for any and all feedback @lcerrato

### CODE REVIEW

File: data/phi0836/phi002/phi0836.phi002.perseus-lat4.xml [hunk 1]
```diff
@@ -18,19 +18,22 @@
...
 						<author>Celsus</author> 
-						<editor>W. G. Spencer</editor> 
+						<editor>Walter George Spencer</editor>
+						<edition>Reprint</edition>
 						<imprint>
 							<pubPlace>Cambridge, Massachusetts</pubPlace> 
 							<publisher>Harvard University Press</publisher> 
-							<date>1971 (Republication of the 1935 edition).</date> 
+							<date>1935-1938</date>
+							<date>1971</date>
```
Comment (lcerrato) [inline data/phi0836/phi002/phi0836.phi002.perseus-lat4.xml:36]: You can type a date, so `<date type="reprint">`

**Review by lcerrato — CHANGES_REQUESTED** [review]

**Comment by capitains-hook** [comment]
*[Hook UI build recap](https://ci.perseids.org/repo/PerseusDL/canonical-latinLit/5355)*

**Comment by AlisonBabeu** [comment]
Hii @lcerrato I will make this change plus I also need to add URLs into the headers.

**Comment by capitains-hook** [comment]
*[Hook UI build recap](https://ci.perseids.org/repo/PerseusDL/canonical-latinLit/5357)*

**Review by lcerrato — APPROVED** [review]

**Pull request merged**